packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -84,8 +84,10 @@ contract OptimisticDistributor is Lockable, MultiCaller, Testable {
     // Merkle Distributor can be set only once.
     MerkleDistributor public merkleDistributor;
```

Comment:
the changes made in this PR: https://github.com/UMAprotocol/protocol/pull/3946#pullrequestreview-990004244 mean that you can likely make this immutable as well!

Reply:
yes, will handle that when merging all audit related PRs